ajax/libs/jquery-ui-bootstrap/package.json

```diff
@@ -2,17 +2,37 @@
   "name": "jquery-ui-bootstrap",
   "description": "A Bootstrap-themed kickstart for jQuery UI widgets",
   "filename": "assets/css/bootstrap.min.css",
-  "version": "0.5pre",
   "keywords": [
     "jquery",
     "bootstrap"
   ],
-  "license": {
-    "name": "MIT License",
-    "url": "http://www.opensource.org/licenses/mit-license.php"
-  },
   "repository": {
     "type": "git",
-    "url": "https://github.com/jquery-ui-bootstrap/jquery-ui-bootstrap.git"
+    "url": "https://github.com/jquery-ui-bootstrap/jquery-ui-bootstrap"
```

Comment:
Let's keep the suffix `.git` here since there is no need to have it updated here.